pronto.gemspec

```
@@ -54,5 +54,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency('rspec-its', '~> 1.2')
   s.add_development_dependency('rspec-expectations', '~> 3.4')
   s.add_development_dependency('rubocop', '~> 0.58')
-  s.add_development_dependency('simplecov', '~> 0.14')
+  # Enforces a simplecov version that is supported by CodeClimate, they support 0.17.x or >= 0.20 (https://docs.codeclimate.com/docs/configuring-test-coverage)
+  s.add_development_dependency('simplecov', '~> 0.17', '!= 0.18.0', '!= 0.18.1', '!= 0.18.2', '!= 0.18.3', '!= 0.18.4',
+                               '!= 0.18.5', '!= 0.19.0', '!= 0.19.1')
```

Considering it's a development-only dependency, I think it's just simpler to do
```suggestion
  s.add_development_dependency('simplecov', '~> 0.20') # see https://docs.codeclimate.com/docs/configuring-test-coverage
```
What do you think?